src/epd_driver/epd_driver.c

```diff
@@ -6,6 +6,8 @@
 #include "esp_log.h"
 #include "esp_types.h"
 
+// Initial rotation: Landscape
+uint8_t display_rotation = 0;
```

Comment:
This variable should be static, since it does not need to be exposed to the outside.
Also, an enum type for the display orientation would be nicer?

Reply:
Sure, can you suggest the right ones? 
The cool thing with an int is that it can be validated just doing <4. This is also the GxEPD way to handle rotation so many people is quite familiar with it.

Reply:
EPD_ROT_LANDSCAPE, EPD_ROT_PORTRAIT, EPD_ROT_INVERTED_LANDSCAPE and EPD_ROT_INVERTED_PORTRAIT?
And in C, enums are really just int ;) So you can just give them the values 0 - 3 and the could would just work when using ints. Enums are just nicer because they hold more semantic information.